Modules/_asynciomodule.c

```diff
@@ -3138,7 +3131,31 @@ static PyObject *
 _asyncio__get_event_loop_impl(PyObject *module, int stacklevel)
 /*[clinic end generated code: output=9c1d6d3c802e67c9 input=d17aebbd686f711d]*/
 {
-    return get_event_loop(stacklevel-1);
+    PyObject *loop;
+    PyObject *policy;
+
+    if (get_running_loop(&loop)) {
+        return NULL;
+    }
+    if (loop != NULL) {
+        return loop;
+    }
+
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
```

Comment:
If we do not need a warning, we can just `return get_event_loop()`.

Reply:
Yeah, now it's the policy that raises the warning. It's a little bit annoying that the policy method is unaware of stacklevel but we can live with that.